Makefile

```diff
@@ -68,6 +69,12 @@ modules: FORCE
 runtime: FORCE
 	cd runtime && $(MAKE)
 
+llvm-runtime-maybe: FORCE
```

Comment:
I actually had named it try-llvm-runtime originally, but then changed the name to llvm-runtime-maybe because it doesn't do the same thing as the other try- targets. In this case, we're not building LLVM or an LLVM runtime and being OK if it fails - instead, we're checking to see if our configuration requires us to build a runtime to support --llvm compilation. So I changed the name to 'llvm-runtime-maybe' as in it might decide it needs a runtime to support --llvm and if it does, that build must succeed... (or at least that's what I want it to do).